girder_worker/utils.py

```diff
@@ -107,14 +107,15 @@ def _flush(self):
                 self._progressCurrent is not None:
             self._redirectPipes(False)
 
-            requests.request(
+            req = requests.request(
                 self.method.upper(), self.url, allow_redirects=True,
                 headers=self.headers, data={
                     'log': self._buf,
                     'progressTotal': self._progressTotal,
                     'progressCurrent': self._progressCurrent,
                     'progressMessage': self._progressMessage
                 })
+            req.raise_for_status()
```

Comment:
I think you're right, just wanted to sanity check, since this gets called during exception handling.